InteractionBase/src/commands/Command.cpp

```diff
@@ -25,13 +25,14 @@
 ***********************************************************************************************************************/
 
 #include "commands/Command.h"
+#include "precompiled.h"
```

Comment:
You never need to include the precompiled header. It's included automatically. Please remove this.